### ISSUE TITLE
Youtube search engine hidden in popup

### ISSUE THREAD

**Issue by simonbcn** [CanisLupus/swift-selection-search#119]
> Arch Linux
> Firefox 62.0.3
> Swift Selection Search 3.27.2

I have added these search engines:
![captura_20181004_135225](https://user-images.githubusercontent.com/428231/46472318-19d6af80-c7dd-11e8-9b36-1e32d6272264.png)

Everything works fine except the Youtube search engine that doesn't appear in the popup and won't let me select it. It only shows a blank space:
![captura_20181004_135153](https://user-images.githubusercontent.com/428231/46472442-7639cf00-c7dd-11e8-86f6-0938c2da1384.png)

In the context menu, it appears and it works well:
![captura_20181004_135359](https://user-images.githubusercontent.com/428231/46472452-80f46400-c7dd-11e8-9aca-997dcc6787ce.png)

**Comment by CanisLupus** [comment]
Hey Simón, have you tried the troubleshooting section in the SSS options page? (inside "Instructions" at the top) I ask this because the last few times people reported the YouTube icon as missing it was due to the adblocker, namely a specific blocking list. :)

I wrote this in the troubleshooting section:
> If an icon does not appear and its URL is correct, please check if that still happens if you disable your ad blocker and add the icon URL again. Some blocking lists in your ad blocker can stop certain images from downloading! (ex.: YouTube may be blocked by "Fanboy's Annoyance List")

That would be the first thing to check. :) If there's still a problem be sure to reply here and we'll try to find the source.

Cheers!
Daniel
